spec/controllers/teams_controller_spec.rb

```diff
@@ -17,6 +17,11 @@
     FactoryBot.build(:team, name: '').attributes
   }
 
+  before(:each) do
+    user = FactoryBot.create(:user)
```

Comment:
Please use consistently either `.create :user` or `.create(:user)`. I'd prefer the option without brackets personally.

Reply:
👍